### DISCUSSION TITLE
🧪 Community Styles

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#19] [General Styles, Prompting, and Theory]
If you would like to post a style or prompt without using a pull request, please use the format below (or a similar format):
Be sure to include the arguments!

**Prompt:** Example
**Arguments:** --version 3 --seed 4776
\<Put Image Here\>

**Comment by dawdawdawdawddawdawdffesf** [comment]
![f1b07d7c-8324-4221-8075-e13f17f076f3](https://user-images.githubusercontent.com/121364264/209453262-13e542ce-90c6-425e-b10d-96e654b3026e.jpg)

**Reply by dawdawdawdawddawdawdffesf** [reply]
> ![f1b07d7c-8324-4221-8075-e13f17f076f3](https://user-images.githubusercontent.com/121364264/209453262-13e542ce-90c6-425e-b10d-96e654b3026e.jpg)

**Reply by willwulfken** [reply]
It took me this long but I think I finally understand. "Community Style"
